themes/default/content/registry/packages/docker-build/image/_index.md

```diff
@@ -0,0 +1,14704 @@
+
+---
+title: "Image"
+title_tag: "docker-build.Image"
+meta_desc: "Documentation for the docker-build.Image resource with examples, input properties, output properties, lookup functions, and supporting types."
+layout: api
+no_edit_this_page: true
+---
+
+
+
+<!-- WARNING: this file was generated by Pulumi Docs Generator. -->
+<!-- Do not edit by hand unless you're certain you know what you are doing! -->
+
+A Docker image built using buildx -- Docker's interface to the improved
+BuildKit backend.
+
+## Stability
+
+**This resource is experimental and subject to change.**
+
+API types are unstable. Subsequent releases _may_ require manual edits
+to your state file(s) in order to adopt API changes.
+
+`retainOnDelete: true` is recommended with this resource until it is
+stable. This enables future API changes to be adopted more easily by renaming
+resources.
+
+Only use this resource if you understand and accept the risks.
+
+## Migrating v3 and v4 Image resources
```

Comment:
It's probably OK to include this all in the resource docs at launch, but I imagine we'll want to move it to a separate guide at some point so this document can explain the resource without basing it off of the user's familiarity with other resources.